hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmMetrics.java

```diff
@@ -156,6 +157,7 @@ public void testVolumeOps() throws IOException {
   }
 
   @Test
+  @Ignore("Test failing because of table cache. Revisit later.")
```

Comment:
Is this difficult to fix?

Reply:
At the very least lets file a blocker jira to fix this test for 0.5.0 so it doesn't get lost.

Reply:
Actually, to say, we can remove these tests, as old code Volume/Bucket/KeyManagerImpl code is not used any more for write requests.

I will open a Jira for this anyway.